tests/spread/integration/apache2-bin/task.yaml

```diff
@@ -0,0 +1,6 @@
+summary: Integration tests for apache2-bin
+
+execute: |
+  rootfs="$(apache2-bin_bins dash_bins)"
+
+  chroot "${rootfs}/" apache2 -v
```

Comment:
Is there any way for us to construct a functional integration test for `apache2` in addition to this smoke test?